src/Momento.Sdk/Auth/ICredentialProvider.cs

```diff
@@ -2,9 +2,21 @@ namespace Momento.Sdk.Auth;
 
 using System.Collections.Generic;
 
+/// <summary>
+/// Interface for credentials
+/// </summary>
 public interface ICredentialProvider
 {
+    /// <summary>
+    /// JWT provided by user
+    /// </summary>
     string AuthToken { get; }
-    string ControlEndpoint {get; }
+    /// <summary>
+    /// Control endpoint
```

Comment:
same comment here; this doesn't give any new information beyond what the name would already provide to the user.

Here I think you might want to say something like "the host and port on which the Momento client will connect to the Momento control plane, in the format '<host>:<port>'" or something. 

(I don't know if that's the actual format, you should base64-decode that segment of your JWT and verify that before updating the docs.